package.json

```diff
@@ -116,6 +117,7 @@
     "prettier": "1.19.1",
     "rollup": "1.29.1",
     "rollup-plugin-commonjs": "10.1.0",
+    "ts-node": "^8.6.2",
```

Comment:
I am weary of adding a dependency for `ts-node`; I think it would be simpler to compile with tsc and run with node than adding an additional pipeline..

Reply:
Fixed!